contracts/SnappAuction.sol

```diff
@@ -132,9 +132,10 @@ contract SnappAuction is SnappBase {
                 )
             );
         }
-        uint currentBatchIndex = standingOrders[accountId].currentBatchIndex;
+        uint32 currentBatchIndex = standingOrders[accountId].currentBatchIndex;
         StandingOrderBatch memory currentOrderBatch = standingOrders[accountId].reservedAccountOrders[currentBatchIndex];
         if (auctionIndex > currentOrderBatch.validFromIndex) {
+            //following line could overflow
```

Comment:
What do you think about using safe math here and throwing in this case?

Reply:
I assume we will eventually make a pass through all contracts and decide in which places safe math is necessary as well.